src/export/StructureDefinitionExporter.ts

```diff
@@ -173,7 +177,19 @@ export class StructureDefinitionExporter implements Fishable {
             if (rule.path !== '') {
               element.setInstancePropertyByPath(rule.caretPath, rule.value, this);
             } else {
-              structDef.setInstancePropertyByPath(rule.caretPath, rule.value, this);
+              try {
+                structDef.setInstancePropertyByPath(rule.caretPath, rule.value, this);
+              } catch (e) {
+                if (e instanceof MismatchedTypeError && rule.isInstance) {
```

Comment:
Sorry I didn't think of this before, but now that I'm looking again, is it possible that here we could just immediately know to defer a rule if it has `IsInstance` true? And then we don't have to try to `setInstancePropertyByPath` and catch the error.

Reply:
You might be correct? This code is intended to handle the situation where `rule.isInstance` is true (which means `rule.value` was parsed from a SEQUENCE) and the rule gets applied successfully. This can happen when the caretPath points at an element with a type that gets handled in `ElementDefinition.fixString`.

Reply:
So the case would be something like this?
```
* ^version = MyInstance
```

That should be an error right? I think that if they want to set a string we should require the usage of "".

Reply:
Ah, I think I understand now. So if we're willing to say "quotes are required when setting a string", then yes, anything caretValueRule with `isInstance` true should get deferred, I think. I was a bit hesitant, but I tried it out with mCODE and nothing broke, so I'll go for it.